doc/classes/AudioEffectHighPassFilter.xml

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectHighPassFilter" inherits="AudioEffectFilter" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Adds a High Pass filter to the Audio Bus.
 	</brief_description>
 	<description>
+		The effect cuts frequences lower than the [code]Cutoff HZ[/code], and lets higher frequences pass.
```

Comment:
- Remove the comma before "and"
- "Cuts frequencies below the [member cutoff_hz]"

Reply:
Shouldn't the Oxford's comma be used, as stated by the [guidelines](http://docs.godotengine.org/en/latest/community/contributing/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything) ?

Reply:
No, that is not an Oxford comma.  "A and B" don't use a comma. "A, B, and C" is when you use the comma.